supabase/migrations/20260824001300_add_vehicle_photos_storage_policies.sql

```
@@ -0,0 +1,82 @@
+-- on it: that was checked in a rolled-back transaction against a local stack
+-- before this file was written, and then by this file applying on a database
+-- rebuilt from nothing. The hosted `postgres` role has the same shape, but only
+-- `db push` there proves it.
+--
+-- tests/unit/storage-bucket-provisioning.test.ts fails if the insert or update
+-- policy for this bucket goes missing or stops naming `auth.uid()`.
+-- tests/e2e/vehicle-photo-upload.spec.ts is the walk: an upload through the
+-- real form against a rebuilt stack, the stored URL fetched with no session,
+-- and an anonymous client, a foreign-folder write and a non-image all refused.
+
+create policy "vehicle-photos: select own"
```

SQLFluff (4.3.0)

[error] Do not use special characters in identifiers.

(RF05)